src/hiero_sdk_python/tck/handlers.py

```
@@ -0,0 +1,44 @@
+import json
+from hiero_sdk_python.tck.protocol import JsonRpcError, PARSE_ERROR, INVALID_REQUEST
+
+_HANDLERS: Dict[str, Any] = {}
+
+def register_handler(method_name: str):
+    def decorator(func):
+        _HANDLERS[method_name] = func
+        return func
+    return decorator
+
+def dispatch(method_name: str, params: Any, session_id: Optional[str]) -> Any:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `params`

(ANN401)